src/ui/public/directives/saved_object_finder.js

```diff
@@ -49,6 +49,22 @@ module.directive('savedObjectFinder', function ($location, $injector, kbnUrl, Pr
       filterResults();
 
       /**
+       * Boolean that keeps track of whether hits are sorted ascending (true)
+       * or descending (false) by title
+       * @type {Boolean}
+       */
+      self.isAscending = true;
+
+      /**
+       * Sorts saved object finder hits either ascending or descending
+       * @param  {Array} hits Array of saved finder object hits
+       * @return {Array} Array sorted either ascending or descending
+       */
+      self.sortHits = function (hits) {
+        return self.isAscending ? _.sortBy(hits, 'title') : _.sortBy(hits, 'title').reverse();
```

Comment:
this should be assigning `self.hits` to the result of the ternary instead of returning it.